CHANGELOG.md

```diff
@@ -8,6 +8,103 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+### Updated
+
+| Component | Helm Chart (s) | App-/KIT Version (s) |
+| :-------- | :-------------: | :------------------: |
+| [BPDM Kit](https://eclipse-tractusx.github.io/docs-kits/category/business-partner-kit) | n/a | [6.0.0](https://eclipse-tractusx.github.io/docs-kits/kits/Business%20Partner%20Kit/Changelog) |
+| [BPN DID Resolution Service (BDRS)](https://github.com/eclipse-tractusx/bpn-did-resolution-service) | [0.5.2](https://github.com/eclipse-tractusx/bpn-did-resolution-service/releases/tag/0.5.2) | [0.5.2](https://github.com/eclipse-tractusx/bpn-did-resolution-service/releases/tag/0.5.2) |
+| [Circularity Kit](https://eclipse-tractusx.github.io/docs-kits/next/kits/Circularity_KIT/page-adoption-view) | n/a | [1.2.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/Circularity_KIT/Changelog%20Circularity%20KIT) |
+| [DCM Kit](https://eclipse-tractusx.github.io/docs-kits/next/category/dcm-kit) | n/a | [1.1.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/DCM-Kit/changelog) |
+| [Digital Product Pass (DPP)](https://github.com/eclipse-tractusx/digital-product-pass) | digital-product-pass: [4.0.1](https://github.com/eclipse-tractusx/digital-product-pass/releases/tag/v4.0.1) | [4.0.1](https://github.com/eclipse-tractusx/digital-product-pass/releases/tag/v4.0.1) |
+| [Digital Twin Kit](https://eclipse-tractusx.github.io/docs-kits/category/digital-twin-kit) | n/a | [2.0.1](https://eclipse-tractusx.github.io/docs-kits/next/kits/Digital%20Twin%20Kit/Digital%20Twin%20Kit%20Changelog) |
+| [Digital Twin Registry](https://github.com/eclipse-tractusx/sldt-digital-twin-registry) | digital-twin-registry: [0.5.2](https://github.com/eclipse-tractusx/sldt-digital-twin-registry/releases/tag/digital-twin-registry-0.5.2) | [0.5.0](https://github.com/eclipse-tractusx/sldt-digital-twin-registry/releases/tag/v0.5.0) |
+| [Discovery Finder](https://github.com/eclipse-tractusx/sldt-discovery-finder) | discoveryfinder: [0.3.1](https://github.com/eclipse-tractusx/sldt-discovery-finder/releases/tag/discoveryfinder-0.3.1) | [0.4.0](https://github.com/eclipse-tractusx/sldt-discovery-finder/releases/tag/v0.4.0) |
+| [Discovery Service (BPN Finder)](https://github.com/eclipse-tractusx/sldt-bpn-discovery) | bpndiscovery: [0.3.1](https://github.com/eclipse-tractusx/sldt-bpn-discovery/releases/tag/bpndiscovery-0.3.1) | [0.4.0](https://github.com/eclipse-tractusx/sldt-bpn-discovery/releases/tag/v0.4.0) |
+| [Eclipse Data Space Connector (Tractus-X EDC)](https://github.com/eclipse-tractusx/tractusx-edc) | [0.7.3](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.3) | [0.7.3](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.3) |
+| [EcoPass Kit](https://eclipse-tractusx.github.io/docs-kits/next/category/eco-pass-kit) | n/a | [1.5.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/Eco_Pass_KIT/changelog) |
```

Comment:
Is it ok that the KITs are in the next version?

Reply:
Or shall we link the correct version even if the links do not work yet?

Reply:
@jSchuetz88 is responsible for the KITs :), But i think you are right. "next" doesn't make sense… it should be a dedicated link.e.g. 24.08 ... but i think this is only possible after the docusaurus Plug-In was executed.

Reply:
How can we run the plugin?

Reply:
> @jSchuetz88 is responsible for the KITs :), But i think you are right. "next" doesn't make sense… it should be a dedicated link.e.g. 24.08 ... but i think this is only possible after the docusaurus Plug-In was executed.

done

Reply:
how much it takes for the tag to be available?

Reply:
@jSchuetz88 already updated the links. They are pointing to the current release, which is now 24.05. But after we executed the versioning plug-in everything should be fine. Please thumbs up ;)

Reply:
You're sure about the version plugin? Previously we didn't use the next versions.

Reply:
Without next it's pointing to the current, with next its pointing to the next (draft) and a specific vesrion is pointing to the related one. Thats what i have in mind :)

Example

- [https://eclipse-tractusx.github.io/docs-kits/kits/Connector%20Kit/page_changelog](https://eclipse-tractusx.github.io/docs-kits/kits/Connector%20Kit/page_changelog) -> current 24.05
- [https://eclipse-tractusx.github.io/docs-kits/24.03/kits/Connector%20Kit/page_changelog](https://eclipse-tractusx.github.io/docs-kits/24.03/kits/Connector%20Kit/page_changelog) -> specific 24.03
- [https://eclipse-tractusx.github.io/docs-kits/next/kits/Connector%20Kit/page_changelog](https://eclipse-tractusx.github.io/docs-kits/next/kits/Connector%20Kit/page_changelog) -> next (work in progress)

Reply:
yep, but the CHANGELOG always should link to the current and not to a next version or versioned artifact. At least we handled it that we the laast time. AFAIK the version plugin doesn't change something.

Reply:
Yep I would also add it directly the version. But in worst case I believe this solution is better than the "next" one

Reply:
Nope, that's a misunderstanding. Don't add a version. Just use the current's link. This link can also be validated.

Just refer to: [rendered github page](https://eclipse-tractusx.github.io/CHANGELOG#2405---2024-05-29) and [this file's older version](https://github.com/eclipse-tractusx/tractus-x-release/blob/main/CHANGELOG.md#2405---2024-05-29).

The changelog needs to be done tomorrow EOB. We are not testing anything new as this may causes time trouble. If we want to change and test things, we can do it WITHIN next release phase prior to INT test. I propose to just stay consistent with recent handling.

Reply:
They will as soon as the version has been successfully committed / merged. Only exception would be unchanged KITs, but these haven't been handled differently in past releases.

Reply:
> yep, but the CHANGELOG always should link to the current and not to a next version or versioned artifact. At least we handled it that we the laast time. AFAIK the version plugin doesn't change something.

thats why we deleted the "next" in the link